SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -159,6 +160,11 @@ + (UIColor *)ows_gray05Color
     return [UIColor colorWithRGBHex:0xEEEFEF];
 }
 
++ (UIColor *)ows_gray15Color
+{
+    return [UIColor colorWithRGBHex:0xD5D6D6];
```

Comment:
Please double-check that Myles still wants this.  I know there's been some churn, but last I heard we _weren't_ going to use gray 15.

Reply:
Will do.

Reply:
Good call - myles confirmed that this should have been gray25